lib/schema/operations_message.graphql

```diff
@@ -0,0 +1,70 @@
+query FindChannelMessages ($filter: ChannelMessageListFilter) {
+    findChannelMessages(filter: $filter) {
+        id
+        createdBy
+        channelId
+        messageText
+        createdAt
+        replyToMessageId
+        deletedBy
+        updatedAt
+        deletedAt
+        editedAt
+        statuses {
+            seenAt
+        }
+    }
+}
+
+query InboxUnseenMessages {
+    myInbox {
+        channels {
+            unseenMessages {
+                channelId
+                id
+                messageText
+                senderFullName
+                createdBy
+            }
+        }
+    }
+}
+
+mutation CreateChannelMessage ($input: ChannelMessageInput!) {
+    createChannelMessage(input: $input) {
+        id
+        createdBy
+        channelId
+        messageText
+        createdAt
+        replyToMessageId
+        deletedBy
+        updatedAt
+        deletedAt
+        editedAt
+        statuses {
+            seenAt
```

Comment:
You need the `userId` as well, otherwise you won't know who saw the message. Might not be critical on 1:1 conversations.

Reply:
Added it, thanks!